Gems/ScriptCanvas/Code/Editor/Assets/ScriptCanvasFileHandling.cpp

```diff
@@ -49,6 +49,35 @@ namespace ScriptCanvasFileHandlingCpp
             }
         }
     }
+
+    class EntityIdMapper
+    {
+    public:
+        EntityIdMapper()
+        {
+            m_newIdsByOld[AZ::EntityId()] = AZ::EntityId();
+            m_newIdsByOld[ScriptCanvas::GraphOwnerId] = ScriptCanvas::GraphOwnerId;
+            m_newIdsByOld[ScriptCanvas::UniqueId] = ScriptCanvas::UniqueId;
```

Comment:
Are these initialization required?

Reply:
Yes, all these values need to be left to their originals.